daktari/file_utils.py

```diff
@@ -42,5 +42,5 @@ def file_contains_text_regex(path: str, regex: str) -> bool:
 
 
 def dir_exists(path: str) -> bool:
-    testing_dir = Path(path)
+    testing_dir = Path(os.path.expandvars(path))
```

Comment:
I guess this is convenient, but it also maybe feels a bit magic that this function would do this automatically? Would it be better to be explicit and use `expandvars` at the call site?

Reply:
Yeah happy to do that instead, not got a strong preference either way

Reply:
I guess this does change things slightly. The failure message previously said "$ENV_VAR_YOU_ARE_CHECKING is not present". After this change it will say "resolved/path/from/that/env/var is not present". Is that worse?

Reply:
![Screenshot 2023-08-09 at 11 45 13](https://github.com/glean-notes/daktari/assets/85498786/6ce8240f-1905-4b93-8ee6-cde491e334de)

Hmm can't see where that `23)` is coming from 🤔 looks like it happens for both versions, so I guess it's existing weirdness in `DirsExist`?

Reply:
Aha it only happens in quiet mode! Looks fine in normal mode

<img width="476" alt="Screenshot 2023-08-10 at 09 54 48" src="https://github.com/glean-notes/daktari/assets/85498786/90336db2-df06-4d34-8304-fb706ddbe29a">

Reply:
I think it's probably fine just showing the path, the suggestion and the message seem nice and clear imo, just wanted to point out that there's a slight behaviour difference